<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="defaultSavePath">
       <property name="text">
        <string>Autodetect Saved Games Folder</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="savePath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="defaultTexturePath">
       <property name="text">
        <string>Autodetect Texture Folder</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="texturePath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="textureBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBrowse</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>browseWorlds()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textureBrowse</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>browseTextures()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultSavePath</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>toggleWorlds(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultTexturePath</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>toggleTextures(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseTextures()</slot>
  <slot>browseWorlds()</slot>
  <slot>toggleTextures(bool)</slot>
  <slot>toggleWorlds(bool)</slot>
 </slots>
</ui>
